core/src/main/java/com/linecorp/armeria/common/MediaTypeNames.java

```diff
@@ -472,6 +476,8 @@
      */
     public static final String ZIP = "application/zip";
 
+    /* multipart types */
+
     /**
      * {@value MULTIPART_FORM_DATA}.
```

Comment:
Not related to this PR, could you place `MULTIPART_FORM_DATA` in alphabetical order together please?

Reply:
I don't know where to put this. 🤣

According to the existing method, it is classified as a `type` unit, but since there was no multipart type, it was added.

Reply:
Ah, it was my mistake. I thought it was sort in alphabetical order. Please ignore this. 😅